mkdocs.yml

```diff
@@ -74,6 +74,7 @@ nav:
           - SPDX: docs/sbom/spdx.md
       - Attestation:
           - SBOM: docs/attestation/sbom.md
+          - Cosign Vulnerability Predicate: docs/attestation/vuln.md
```

Comment:
To align with their doc.

>Cosign Vulnerability Scan Record Attestation Specification

https://github.com/sigstore/cosign/blob/main/specs/COSIGN_VULN_ATTESTATION_SPEC.md

```suggestion
          - Cosign Vulnerability Scan Record: docs/attestation/vuln.md
```